packages/infoportal-scripts/src/kobo/BuildTypeKobo.ts

```diff
@@ -365,6 +365,9 @@ export class BuildKoboType {
       meal_verificationPartnerBnre: {
```

Comment:
Changes from this file should have been in a dedicated commit/PR since they could have been tricky. But nevermind since it's done, it's ok.